tutorials/nlp/Entity_Linking_Medical.ipynb

```diff
@@ -179,6 +185,7 @@
     "cfg.model.nemo_path = os.path.join(PROJECT_DIR, \"tiny_example_sap_bert_model.nemo\")\n",
     "cfg.model.train_ds.data_file = os.path.join(DATA_DIR, \"tiny_example_train_pairs.tsv\")\n",
     "cfg.model.validation_ds.data_file = os.path.join(DATA_DIR, \"tiny_example_validation_pairs.tsv\")\n",
+    "cfg.trainer.amp_backend = \"apex\"\n",
```

Comment:
Why is this here? By default we use `native` for PyTorch native amp.

Reply:
Oh I see now from the bug. Just remove `amp_level` and `amp_backend`. The defaults should work fine.